src/components/NewWorkspaceModal.js

```diff
@@ -260,12 +242,12 @@ const NewWorkspaceModal = withDisplayName('NewWorkspaceModal', ({
           onChange: ({ value }) => setNamespace(value),
           styles: { option: provided => ({ ...provided, padding: 10 }) },
           options: _.map(({ projectName, invalidBillingAccount, cloudPlatform }) => ({
-            'aria-label': `${getCloudPlatformTitle(cloudPlatform)} ${projectName}${ariaInvalidBillingAccountMsg(invalidBillingAccount)}`,
+            'aria-label': `${cloudProviderLabels[cloudPlatform]} ${projectName}${ariaInvalidBillingAccountMsg(invalidBillingAccount)}`,
             label: h(TooltipTrigger, {
               content: invalidBillingAccount && invalidBillingAccountMsg, side: 'left'
             },
             [div({ style: { display: 'flex', alignItems: 'center' } },
-              [h(cloudPlatformIcon, { cloudPlatform, key: projectName }), projectName]
+              [h(CloudProviderIcon, { key: cloudPlatform, cloudProvider: cloudPlatform, style: { marginRight: '0.5rem' } }), projectName]
```

Comment:
I'm confused about the key here-- it won't be unique, which I thought was the point of having it before.

Reply:
It doesn't look like a key should be necessary here, so I removed it. And then React complained about it, so I added it back, but mistakenly used cloudPlatform instead of projectName.

Reply:
Then again... either should work since the only two children here are the icon and the project name string. I still don't understand why a key is needed here.